src/main/resources/l10n/Menu_de.properties

```diff
@@ -56,7 +56,7 @@ Paste=&Einfügen
 Preferences=&Einstellungen
 Previous_tab=&Vorheriger_Tab
 Quit=&Beenden
-Recent_files=&Zuletzt_geöffnete_Dateien
+Recent_databases=
```

Comment:
Please add a German translation as I think this was the whole idea of the linked pr.